docs/examples/en/controls/OrbitControls.html

```diff
@@ -91,7 +91,7 @@ <h2>Properties</h2>
 		<h3>[property:Boolean autoRotate]</h3>
 		<p>
 			Set to true to automatically rotate around the target.<br> Note that if this is enabled, you must call [page:.update]
-			() in your animation loop.
+			() in your animation loop. If you want the animation to be frame rate independent, apply a time delta value to [page:.update]().
```

Comment:
How about something like this, instead...

If you want the auto-rotate speed to be independent of the frame rate (the refresh rate of the display), you must pass the time `deltaTime`, in seconds, to page.update().

Reply:
That sounds good to me! Do you mind updating the doc pages with your suggestion?